packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -214,17 +216,23 @@ class Splitter extends CollectionWidget<Properties> {
 
   _attachHoldEvent(): void {}
 
-  _resizeHandler(): void {
-    if (!this._shouldRecalculateLayout) {
-      return;
-    }
+  _isAttached(): boolean {
+    return !!contains(domAdapter.getBody(), $(this.element()).get(0));
```

Comment:
Hm, does it work in ShadowDOM? I mean if popup content is a shadow root

Reply:
That's why I opted for the implementation using the utility, and this approach is already in use. If it doesn't work here, it likely won't work there either, which is improbable. It used in [template_base.js](https://github.com/EugeniyKiyashko/DevExtreme/blob/24_2_splitter_bug/packages/devextreme/js/core/templates/template_base.js#L37-L38)

Reply:
Oh, yeah, we have shadowDOM support in the `contains` method